User interface text update

bonnaphil opened the pull request:
User interface text update

bonnaphil committed:
User interface text update

User interface text update

MarkDHamill commented:
I don't think I should merge this, but if I do it should go on the 3.0.7 branch. I will be updating the README.md at the appropriate point to reference your French translation and Praggle's German translation.

The extension review team requires only British English in the archive they review and approve.

bonnaphil committed:
French update for Digests 3.2.0 (RC7)

This version is updated for Digests 3.2.0 (RC7) and corrects some
localization issues.
Version mise à jour pour Digests 3.2.0 (RC7) + correction de quelques
coquilles de localisation.

bonnaphil committed:
Merge branch 'MarkDHamill/master' into fr
